src/main/index.ts

```diff
@@ -126,3 +155,31 @@ export function extractFeatureMainThread(
     options
   );
 }
+
+export function extractFeatureObservable({
```

Comment:
Should there be a separate implementation of this library that does observable? Maybe...

Reply:
The thinking here was that rxjs might bump the bundle size too much to be acceptable for a use case where you don't need streaming.